birdhouse/optional-components/scheduler-job-autodeploy/config.yml.template

```diff
@@ -9,11 +9,10 @@
     --volume ${COMPOSE_DIR}/..:${COMPOSE_DIR}/..:rw
     --volume ${BIRDHOUSE_AUTODEPLOY_DEPLOY_KEY_ROOT_DIR}:${BIRDHOUSE_AUTODEPLOY_DEPLOY_KEY_ROOT_DIR}:ro
     --volume ${BIRDHOUSE_SSL_CERTIFICATE}:${BIRDHOUSE_SSL_CERTIFICATE}:ro
-    --volume ${BIRDHOUSE_LOCAL_ENV_REAL_PATH}:/tmp/birdhouse-local-env
```

Comment:
What about:

```
--volume ${BIRDHOUSE_LOCAL_ENV_REAL_PATH}:${BIRDHOUSE_LOCAL_ENV_REAL_PATH}
--volume ${BIRDHOUSE_LOCAL_ENV}:${BIRDHOUSE_LOCAL_ENV}
--env BIRDHOUSE_LOCAL_ENV=${BIRDHOUSE_LOCAL_ENV}
```

Reply:
This will likely not work because originally it had `--volume ${BIRDHOUSE_LOCAL_ENV_REAL_PATH}:${BIRDHOUSE_LOCAL_ENV_REAL_PATH}` and I had to remove it because we can not double volume-mounting the folder containing `env.local` and `env.local` itself and keep write-access (readonly-access is fine) with the newer docker image, see https://github.com/bird-house/birdhouse-deploy/pull/597#issuecomment-3426793138 

But I could give `--volume ${BIRDHOUSE_LOCAL_ENV}:${BIRDHOUSE_LOCAL_ENV}` a try, although I expect the same result since the symlink points to the same place anyways.